### ISSUE TITLE
ZAP Open Data QAQC 

### ISSUE THREAD

**Issue by AmandaDoyle** [NYCPlanning/db-zap-opendata#22]
- [ ] Do we need to rename `dcp_applicanttype` to `applicant_type` in the projects table?

**Comment by SashaWeinstein** [comment]
This is a good question, applicant type breaks the pattern of all the other columns. The SQL to switch the field names was written before I started so the open data has always had these fields names. Easy to switch over, needs to be changed in 2 places, SQL query and recode process

**Comment by croswell81** [comment]
yes, it should be **applicant_type** according to the data dictionary. Will review all field names once I confirm access to the data.
